<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MCUUpgrade</class>
 <widget class="QWidget" name="MCUUpgrade">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MCUUpgrade {
background: #FFFFFF;
border-radius: 10px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="grpBox_9">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    border: 1px solid rgb(200, 200, 200);
    font-size: 24px;
    font-family: &quot;PingFang SC&quot;;
    border-radius: 5px;
    margin-top: 2ex;
	font-weight: 500;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    left: 0px;
    margin-left: 0px;
    padding: 0 1px;
	color: #666666;
}</string>
     </property>
     <property name="title">
      <string>下位机升级</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>120</horstretch>
            <verstretch>40</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">width: 93px;
height: 23px;
font-family: PingFang SC;
font-weight: 500;
font-size: 24px;
color: #666666;
line-height: 35px;</string>
          </property>
          <property name="text">
           <string>手动连接</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="com_port">
          <property name="minimumSize">
           <size>
            <width>700</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox {
    border: 1px solid #DCDEE0;
    border-radius: 6px;
    font-family: &quot;PingFang SC Medium&quot;;
    font-size: 24px;
    padding-left: 10px;
    background-color: rgb(250, 251, 252);
	font-weight: 500;
    color: #000000;
}

QComboBox::drop-down {
    margin: 0px;
    padding-right: 20px;
    border: 0px;
}

QComboBox::down-arrow {
    image: url(:/static/PHOTOS/combo_up.png);
}

QComboBox::down-arrow:on {
    image: url(:/static/PHOTOS/combo_down.png);
}

QListView {
    margin: 0px;
    padding: 0px;
    border: 0px;
    border-radius: 10px;
    outline: 0px;
    font-size: 24px;
    color: #333333;
    background-color: rgb(250, 251, 252);
}

QListView::item {
    margin: 0px;
    padding: 10px;
    border: 0px;
    border-radius: 0px;
    height: 60px;
}

QListView::item:hover {
    color: #333333;
}

QListView::item:selected {
    color: rgb(31, 112, 227);
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_link">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background: #ffaa00;
border-radius: 10px;

font-family: PingFang SC;
font-weight: bold;
font-size: 28px;
color: #FEFEFE;
line-height: 42px;</string>
          </property>
          <property name="text">
           <string>连接尝试</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>120</horstretch>
            <verstretch>40</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">width: 93px;
height: 23px;
font-family: PingFang SC;
font-weight: 500;
font-size: 24px;
color: #666666;
line-height: 35px;</string>
          </property>
          <property name="text">
           <string>升级设备</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="com_device">
          <property name="minimumSize">
           <size>
            <width>700</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox {
    border: 1px solid #DCDEE0;
    border-radius: 6px;
    font-family: &quot;PingFang SC Medium&quot;;
    font-size: 24px;
    padding-left: 10px;
    background-color: rgb(250, 251, 252);
	font-weight: 500;
    color: #000000;
}

QComboBox::drop-down {
    margin: 0px;
    padding-right: 20px;
    border: 0px;
}

QComboBox::down-arrow {
    image: url(:/static/PHOTOS/combo_up.png);
}

QComboBox::down-arrow:on {
    image: url(:/static/PHOTOS/combo_down.png);
}

QListView {
    margin: 0px;
    padding: 0px;
    border: 0px;
    border-radius: 10px;
    outline: 0px;
    font-size: 24px;
    color: #333333;
    background-color: rgb(250, 251, 252);
}

QListView::item {
    margin: 0px;
    padding: 10px;
    border: 0px;
    border-radius: 0px;
    height: 60px;
}

QListView::item:hover {
    color: #333333;
}

QListView::item:selected {
    color: rgb(31, 112, 227);
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>120</horstretch>
            <verstretch>40</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">width: 93px;
height: 23px;
font-family: PingFang SC;
font-weight: 500;
font-size: 24px;
color: #666666;
line-height: 35px;</string>
          </property>
          <property name="text">
           <string>升级文件</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>37</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_path">
          <property name="minimumSize">
           <size>
            <width>700</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">width: 688px;
height: 60px;
background: #FFFFFF;
border-radius: 10px;
border: 1px solid rgba(170,189,212,0.6);

padding-left: 10px;

font-family: PingFang SC;
font-weight: 500;
font-size: 24px;
color: #333333;
line-height: 35px;</string>
          </property>
          <property name="text">
           <string>选择文件</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_upgrade">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>62</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>PingFang SC</family>
            <pointsize>-1</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background: #437AFF;
border-radius: 31px;

font-family: PingFang SC;
font-weight: bold;
font-size: 28px;
color: #FEFEFE;
line-height: 42px;</string>
          </property>
          <property name="text">
           <string>下位机升级</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
